### PR TITLE
Add Nuri Scroll to dex.trades

### PR DESCRIPTION
**Pull request by Jam516**
Add Nuri Scroll to dex.trades

### CODE REVIEW

File: models/_sector/dex/trades/scroll/platforms/nuri_scroll_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v3_trades(
+        blockchain = 'scroll',
+        project = 'nuri',
```
Comment (Hosuke) [inline models/_sector/dex/trades/scroll/platforms/nuri_scroll_base_trades.sql:16]: Hi @Jam516,
we may need to add `nuri` into this `dex.info`:
https://github.com/duneanalytics/spellbook/blob/main/models/dex/dex_info.sql

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/_sector/dex/trades/scroll/platforms/nuri_scroll_base_trades.sql (hunk 1 above)
Comment (Jam516) [reply models/_sector/dex/trades/scroll/platforms/nuri_scroll_base_trades.sql:16]: Thanks! Added it now

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Jam516 

[Sample nuri_scroll_base_trades](https://dune.com/queries/3772498)

**Review by jeff-dude — APPROVED** [review]
thank you!

**Comment by Jam516** [comment]
Hi @jeff-dude , ready for merging here

**Pull request merged**